lib/bloc/user_profile/user_profile_bloc.dart

```diff
@@ -160,6 +163,22 @@ class UserProfileBloc {
     return user;
   }
 
+  _updateBiometricsSettings() {
+    _checkBiometrics();
+    _deviceService.eventStream.listen((e){
+      if (e == NotificationType.RESUME) {
+        _checkBiometrics();
+      }
+    });
+  }
+
+  Future _checkBiometrics() async {
+    String enrolledBiometrics = await _localAuthService.enrolledBiometrics;
+    _updateSecurityModel(UpdateSecurityModel(_currentUser.securityModel.copyWith(
```

Comment:
Just to make sure this means that we reset the persistent user selection to "not use fingerprint" in case the device settings has changed. Is this the requirement?

Reply:
Yes, this resets the user setting.

It prevents users to bypass app security by defining a new fingerprint.

1- User A has fingerprint auth enabled on app
2- User A deletes pin/fingerprint from device(for some reason)
3- User A's device gets into hand of a person with malicious intent, User B
4- User B opens Breez App and is required to enter pin 

If we didn't reset this setting, User B could define a new fingerprint and unlock the Breez App with that newly defined fingerprint.

Reply:
I actually don't read the code this way. The code resets the setting in case there is no fingerprint defined in the device.
In the case you wrote the requirement was to reset in case there was a **change** in the fingerprint but it is still defined (by user B).

Reply:
We are resetting the setting on stage 4, if we didn't the user could define a new fingerprint and bypass app security.
It is still possible to bypass app security by defining the new fingerprint before opening the app but we will detect the change in fingerprint and prevent this altogether.